Annotato/Annotato/Persistence/PersistenceService/OnlinePersistenceService/OnlinePersistenceService+DocumentSharesPersistence.swift

```diff
@@ -2,12 +2,14 @@ import AnnotatoSharedLibrary
 import Foundation
 
 extension OnlinePersistenceService: DocumentSharesPersistence {
-    func createDocumentShare(documentShare: DocumentShare) async -> DocumentShare? {
-        guard let createdDocumentShareRemote = await remotePersistence
+    func createDocumentShare(documentShare: DocumentShare) async -> Document? {
+        guard let sharedDocument = await remotePersistence
             .documentShares
             .createDocumentShare(documentShare: documentShare) else {
             return nil
         }
-        return createdDocumentShareRemote
+
+        _ = await localPersistence.documents.createDocument(document: sharedDocument)
```

Comment:
Let's return the result from here for consistency with `OnlinePersistenceService+DocumentsPersistence`. Unless there is a good reason to do it this way.